test/testData/ePO-CM-v2.0.1-2020-03-27_test.eap.xmi

```diff
@@ -27218,7 +27218,7 @@
 						<tags/>
 						<xrefs value="$XREFPROP=$XID={34B58188-7356-4980-9E42-5BCF7D94FAEC}$XID;$NAM=Stereotypes$NAM;$TYP=attribute property$TYP;$VIS=Public$VIS;$PAR=0$PAR;$DES=@STEREO;Name=disjointWithFixedValue;GUID={45689949-3445-4130-9146-7F27285C8C4E};@ENDSTEREO;$DES;$CLT={E11C20C1-6FAD-4a4d-AB89-5D1BAF1DDFB2}$CLT;$SUP=&lt;none&gt;$SUP;$ENDXREF;"/>
 					</attribute>
-					<attribute xmi:idref="EAID_70C20D04_6B23_42c2_A79D_EA9EA0CE540F" name="ep!o:1Weight ValueType*" scope="Public">
+					<attribute xmi:idref="EAID_70C20D04_6B23_42c2_A79D_EA9EA0CE540F" name="epo:WeightValueType" scope="Public">

```

Comment:
I believe that `ep!o:1Weight ValueType*` is modified on purpose to create test data for a wrong/unacceptable element name. 
To overcome this problem, please select a recent XMI, not a v2.0.1 one. 

If you make this correction, probably, some unit tests will not pass anymore. Please double-check.